docs/content/how-mixins-work.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Mixins 
+descriptions: How do mixins work?
+---
+
+## What is a Mixin
+
+A cloud native application bundle built with Porter consists of two main components: the Porter runtime and a declarative manifest file named `porter.yaml`.
```

Comment:
I think this paragraph belongs in that other page to be created, that explains porter in general, and there are 3 parts: buildtime, runtime and the manifest.

Reply:
Let's make sure to explain what buildtime and runtime is.

buildtime = when we build the invocation image
runtime = when the user is execution actions against the invocation image, such as install